resources/assets/components/utilities/CtaPopover/CtaPopoverEmailForm.js

```diff
@@ -36,8 +36,10 @@ const CtaPopoverEmailForm = ({ handleComplete }) => {
       })
       .catch(error => {
         report(error);
-        const errorMessage =
-          get(error, 'response.error.fields.email', [])[0] || error.response;
+        const errorMessage = isString(error.response)
+          ? error.response
+          : get(error, 'response.error.fields.email', [])[0] ||
```

Comment:
This might be a nice place to use Lodash's `first` method:

```suggestion
          : first(get(error, 'response.error.fields.email'))
```

Reply:
Sweet! @DFurnes how does the `first` method work?

Reply:
It's an alias for Lodash's [`head`](https://lodash.com/docs/4.17.15#head) method (which IMO is a little less clearly named). Either way, it'll just grab the first element from the array (or return `undefined` if passed a non-array).

Reply:
ahhhhh I love that! thanks @DFurnes